EIPS/eip-2028.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 2028
+title: Calldata gas cost reduction
+author: Alexey Akhunov(@AlexeyAkhunov), Eli Ben Sasson (eli@starkware.co), Tom Brand (tom@starkware.co), Avihu Levy (avihu@starkware.co)
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-03
+---
```

Comment:
Can you please add a `discussion-url`? A lot of authors use https://ethereum-magicians.org to open a topic, but this is not a requirement. The only restriction is that it cannot point to a pull request.